Reading List/Reading List/View Controllers/BookDetailViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  BookDetailViewController.swift
+//  Reading List
+//
+//  Created by Ian Becker on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailViewController: UIViewController {
+    
+    var bookController: BookController?
+    var book: Book?
+    
+    @IBOutlet weak var bookTitle: UITextField!
+    @IBOutlet weak var reasonToRead: UITextView!
+    @IBAction func saveBook(_ sender: UIBarButtonItem) {
```

Comment:
Lets think what should happen when the user presses this button. What is the user expecting to happen here? What action do we need to code/program?